docs/understanding-tracking-design/managing-your-data-structures/ui/index.md

```diff
@@ -14,65 +12,36 @@ To create a new [Data Structure](/docs/understanding-your-pipeline/schemas/index
 
 ![](images/image-1.png)
 
-Now select whether you'd like to create an [Event](/docs/understanding-your-pipeline/events/index.md) or an [Entity](/docs/understanding-your-pipeline/entities/index.md). You can always change this selection at a later date.
-
-![](images/image-2.png)
+When creating a new data structure, you can select from two options: **Create using builder** or **Create using JSON**.
```

Comment:
Only in Enterprise, right?

Reply:
Also in this sentence we should give the guidance on which method is recommended when. You can more or less copy the info box from the Builder page.